pkg/webhook/managedresource/validatingadmissionpolicy.go

```diff
@@ -63,13 +63,13 @@ func mutateValidatingAdmissionPolicy(vap *admv1.ValidatingAdmissionPolicy) {
 				    &&
 					(
 						"system:masters" in request.userInfo.groups ||
-						"system:serviceaccounts:kube-system" in request.userInfo.groups ||
 						"system:serviceaccounts:fleet-system" in request.userInfo.groups
 					)
 				)
 				  ||
 				(
-					"system:serviceaccounts:openshift-infra" in request.userInfo.groups
+					"system:serviceaccounts:openshift-infra" in request.userInfo.groups ||
+					"system:serviceaccounts:kube-system" in request.userInfo.groups
```

Comment:
instead of exempt all kube-system group, we should exempt "namespace-controller" instead